tests/Unit/Evolution/Test_ComputeTags.cpp

```diff
@@ -0,0 +1,51 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "tests/Unit/TestingFramework.hpp"
+
+#include <pup.h>
+#include <string>
+
+#include "DataStructures/DataBox/DataBox.hpp"
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "DataStructures/DataBox/Prefixes.hpp"
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "DataStructures/Variables.hpp"
+#include "Evolution/ComputeTags.hpp"
+#include "Time/Tags.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace {
+
+struct FieldTag : db::SimpleTag {
+  using type = Scalar<DataVector>;
+};
+
+struct AnalyticSolution {
+  tuples::TaggedTuple<FieldTag> variables(
+      const tnsr::I<DataVector, 1>& x, const double t,
+      const tmpl::list<FieldTag> /*meta*/) const noexcept {
+    return {Scalar<DataVector>{t * get<0>(x)}};
+  }
+  void pup(PUP::er& /*p*/) noexcept {}  // NOLINT
+};
+
+struct AnalyticSolutionTag : db::SimpleTag {
+  using type = AnalyticSolution;
+};
+
+}  // namespace
+
+SPECTRE_TEST_CASE("Unit.Evolution.ComputeTags", "[Unit][Evolution]") {
+  tnsr::I<DataVector, 1, Frame::Inertial> inertial_coords{{{{1., 2., 3., 4.}}}};
```

Comment:
could you add a check of the `tag_name` of the `analytic_compute` tag?

Reply:
ping

Reply:
Oh, I misread and added the (also missing) test for the prefix tag. Also added this test now.